packages/components/psammead-navigation/src/index.stories.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
```

Comment:
Could we add tests and stories for the AMP components?

Reply:
I was not sure about adding AMP stories since they will look exactly the same as Canonical ones. Will this be useful?

With regards to tests, if I add snapshots for AMP, they will be identical to Canonical navigation. Since the only difference of `on` vs `onChange` is not affecting appearance and thus is not registered in snapshots. So again, was not sure if it's worth adding an AMP version?

Reply:
I think we should definitely have a test for the AMP version - even if it looks the same, they're supposed to look the same, and this will guard against accidentally breaking something if changes are made to one but not to the other.

Reply:
Makes sense, I will add a test then. What do you think about AMP stories though?

Reply:
I also think an AMP story would be useful - you can create everything that needs to be passed into the component once and then pass it into the canonical wrapper and then into the AMP wrapper?